sound/soc/sof/Kconfig

```diff
@@ -191,6 +191,16 @@ config SND_SOC_SOF_DEBUG_IPC_FLOOD_TEST
 	  Say Y if you want to enable IPC flood test.
 	  If unsure, select "N".
 
+config SND_SOC_SOF_DEBUG_IPC_FLOOD_TEST_CLIENT
+	tristate "SOF enable IPC flood test client"
```

Comment:
careful with tristates, is there a need for the client to be compiled in the same way as the parent? If we provide independent choices then there's a risk for errors.

Reply:
ignore this comment, depends_on forces the same as the parent.